Changes

```diff
@@ -94,6 +94,9 @@ Next version (4.05.0):
 
 ### Compiler distribution build system:
 
+- GPR#887: allow -with-frame-pointers if clang is used as compiler
+  (Bernhard Schommer)
```

Comment:
Could you specify "as compiler on Linux"? Otherwise it could confuse MacOS or Windows users.

(Then it's good for merging.)